etc/ganeti/instance-debootstrap/variants/bookworm.conf

```diff
@@ -0,0 +1,64 @@
+# ganeti-instance-debootstrap defaults file
+
+# if you want to change from the default of installing debian stable
+# on the next instance, customize this file before the instance
+# installation
+
+# PROXY: if non-null, use this as an http(s)-proxy in order to speed
+# up non-cached installs or provide internet access if not directly
+# possible; not that if not set, debootstrap might still use a
+# system-wide proxy setting if it is exported in the ganeti-noded
+# daemon environment (but the node daemon environment is cleaned up
+# and not exported starting with Ganeti 2.5)
+#PROXY="http://10.0.2.162:8000/"
+
+# MIRROR: do not customize MIRROR if you want to be able to install
+# both debian and ubuntu, since they have different defaults; or
+# customize it before each install
+MIRROR="http://ftp.us.debian.org/debian"
```

Comment:
Maybe use deb.debian.org here to avoid getting unlucky with a slow mirror?

Reply:
Oh yea, the Fastly mirror is pretty good.